docs/HowTo/Develop-Dapps/Use-web3js.md

```diff
@@ -1,59 +1,72 @@
-description: Hyperledger Besu client libraries 
-<!--- END of page meta data -->
+---
+description: Hyperledger Besu client libraries
+---
 
-# Create a Signed Transaction using Web3.js
+# Create a signed transaction using Web3.js
 
-You can use the example Javascript scripts to create and send raw transactions in the private network 
-created by the [Private Network Example](../../Tutorials/Examples/Private-Network-Example.md).
+You can use the example JavaScript scripts to create and send raw transactions in the private
+network created by the
+[Private Network Example](../../Tutorials/Examples/Private-Network-Example.md).
 
-Update the `JSON-RPC endpoint` in the following examples to the endpoint for the private 
-network displayed after running the `./run.sh` script.
+In the following examples update the `JSON-RPC endpoint` to the endpoint for the private network
+displayed after running the `./run.sh` script.
 
 !!! warning "Private keys"
 
     Do not use the accounts in the examples on mainnet or any public network except for testing.
-    The private keys are displayed which means the accounts are not secure.
+    They display the private keys, which means the accounts are not secure.
     
```

Comment:
`./docs/HowTo/Develop-Dapps/Use-web3js.md:17: MD009 Trailing spaces`
line number 17 is missleading as the spaces are really on the line 18.

Reply:
I could have sworn I fixed those before. hmmmm. Fixed now.